### DISCUSSION TITLE
SSO integration

### DISCUSSION THREAD

**Discussion by gardiol** [photoview/photoview#1034] [General]
There are a few oldish tickets on this, but i think it's better to consolidate in a discussion.

I am using Authelia as SSO, and it would be great if photoview could inherit the authentication from it.

Also, another proposed possibility was to fully disable login / authentication in PhotoView and let upstream manage it (de facto a single-user mode)

I think that an approach with proxy headers like Navidrome does, would be easy and good to have.

**Comment by kkovaletp** [comment]
@gardiol, thanks for raising this topic up.
In general, I'd like to have an SSO/External auth provider integration with no vendor lock, so users could integrate their preferred provider.

@jordy2254 was working on the Auth topic, so I'd like to let him review this case and provide his thoughts

**Reply by gardiol** [reply]
Fully agree with you. Header authentication for example is well supported by all providers, as well adls basic authentication and of course OIDC...
